### DISCUSSION TITLE
Problems with Google Sign In Prompt

### DISCUSSION THREAD

**Discussion by grinlap** [Cookie-AutoDelete/Cookie-AutoDelete#1479] [Q&A / Support]
For some time now I've been getting a pop up labeled "Sign in with Google" "Use your Google Account to sign in to <the web site I'm on>". I searched for how to get rid of this annoying pop up and found that I could change a setting in my Google account to prevent this. (See https://www.howtogeek.com/735152/how-to-turn-off-the-sign-in-with-google-prompt-on-websites/.) I changed the setting but it didn't stop the pop up. I suspected that the problem might be related to cookies so I checked the CAD expression list and found *.google.com. Noticing that the offending pop up was in a separate iframe I opened it in a new tab to see if I could get its URL. That turned out to be accounts.google.com which should have been included in *.google.com I thought. I added accounts.google.com to the expression list just in case but no luck. However, if I uncheck Enable Automatic Cleaning the pop up no longer appears. 

I can't be specific about how long this problem has been present but my impression is a few weeks or so. I'm using CAD 3.8.2 which was updated on Dec 12, 2022. Was this problem introduced with this update??? I'm using Firefox and this problem has been present through at least v107 & 108.

So, color me confused. The problem occurs on many web sites, seems to be related to cookies but is not cured when I white list *.google.com or accounts.google.com.

CAD 3.8.2, Windows 10 latest updates, Firefox 108.0.1

Any help appreciated.

**Comment by kennethtran93** [comment]
Are you using any containers of any kind in Firefox and if so did you also enable cleanup of any other browsing data type items?

If so, and you use google on other containers but do not whitelist that in those containers, then the other browsing data types would get removed (except cookies) due to the way the webextension API was first available (it wasn't available to be filtered by container - all we could do at that time was pass in domains to clean up for each of the other browsing data types and the API would clean up everything from that domain in all containers).

I'm not entirely sure how google is storing that specific data (either cache, localstorage, indexeddb, service workers, etc).

accounts.google.com would be included under *.google.com

**Reply by grinlap** [reply]
Kenneth,
No containers. I'm using Ghostery and uBlock Origin. Exploring further showed that trusting the site in Ghostery (not Google but the site I'm actually on) will prevent the Google pop up form appearing. By specifically trusting ONLY the Google tracker (Ghostery's term) I can prevent the pop up on the particular site. The pop up doesn't appear on every site I visit so I'm going to try and determine if its appearance is related to the Ghostery settings. 

Thanks for the reply, it got me looking further afield than CAD. The problem seems to be related to a Google tracker, perhaps not a cookie as such, but I can't blame CAD at this point. I'm going to collect more info.

**Reply by kennethtran93** [reply]
Maybe Ghostery was also blocking and/or removing the relevant cookie/site data that houses the value that hides the google sign in...
